image.go

```diff
@@ -27,7 +27,7 @@ func publishImages(vals []interface{}, receipt msgReceipt) []interface{} {
 }
 
 // publishImages sends a "display_data" broadcast message for given image.
-func publishImage(img image.Image, receipt msgReceipt) error {
+func publishImage(img image.Image, receipt *msgReceipt) error {
 	bytes, mime, err := encodePng(img)
 	if err != nil {
 		return err
```

Comment:
These functions can move into the same package for rendering and ideally keep them more or less pure. This separation will be really helpful both for testing and maintaining.